### PR TITLE
Modify utility_functions reader() implementation to be more permissive

### PR DESCRIPTION
**Pull request by joschock**
Modify utility_functions reader() implementation to be more permissive in the face of unexpected encoding coming from utilities executed with RunCmd()

### CODE REVIEW

**Review by spbrogan — CHANGES_REQUESTED** [review]
How about making it an optional requirement to runcmd.  Default would be error as it is now.   But this would allow a caller to ignore the errors if so desired.

**Comment by joschock** [comment]
> How about making it an optional requirement to runcmd. Default would be error as it is now. But this would allow a caller to ignore the errors if so desired.

Done.

File: edk2toollib/utility_functions.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def wrap(*args):
 # @return returncode of called cmd
```
Comment (spbrogan) [inline edk2toollib/utility_functions.py:166]: Need a function header comment for the variable...but otherwise looks good.

Comment (joschock) [reply edk2toollib/utility_functions.py:168]: Thanks, added.

**Review by corthon — APPROVED** [review]

**Pull request merged**